src/main/java/com/atlauncher/gui/tabs/VanillaPacksTab.java

```diff
@@ -233,177 +185,69 @@ private void checkDirty(DocumentEvent e) {
         gbc.anchor = GridBagConstraints.BASELINE_LEADING;
-
-        JScrollPane minecraftVersionScrollPane = new JScrollPane(JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED,
-                JScrollPane.HORIZONTAL_SCROLLBAR_NEVER);
-
+        JScrollPane minecraftVersionScrollPane = new JScrollPane(
+            JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED, JScrollPane.HORIZONTAL_SCROLLBAR_NEVER
+        );
         minecraftVersionScrollPane.setPreferredSize(new Dimension(450, 300));
-
         setupMinecraftVersionsTable();
-        reloadMinecraftVersionsTable();
         minecraftVersionScrollPane.setViewportView(minecraftVersionTable);
-
         mainPanel.add(minecraftVersionScrollPane, gbc);
 
         // Loader Type
         gbc.gridx = 0;
         gbc.gridy++;
         gbc.insets = UIConstants.LABEL_INSETS;
         gbc.anchor = GridBagConstraints.EAST;
-
         JLabel loaderTypeLabel = new JLabel(GetText.tr("Loader") + "?");
         mainPanel.add(loaderTypeLabel, gbc);
-
         gbc.gridx++;
         gbc.insets = UIConstants.FIELD_INSETS;
         gbc.anchor = GridBagConstraints.BASELINE_LEADING;
-
         loaderTypeButtonGroup.add(loaderTypeNoneRadioButton);
         loaderTypeButtonGroup.add(loaderTypeFabricRadioButton);
         loaderTypeButtonGroup.add(loaderTypeForgeRadioButton);
         loaderTypeButtonGroup.add(loaderTypeLegacyFabricRadioButton);
         loaderTypeButtonGroup.add(loaderTypeQuiltRadioButton);
-
         JPanel loaderTypePanel = new JPanel(new FlowLayout());
-        loaderTypePanel.add(loaderTypeNoneRadioButton);
 
-        if (ConfigManager.getConfigItem("loaders.fabric.enabled", true) == true) {
-            loaderTypePanel.add(loaderTypeFabricRadioButton);
-        }
-
-        if (ConfigManager.getConfigItem("loaders.forge.enabled", true) == true) {
-            loaderTypePanel.add(loaderTypeForgeRadioButton);
-        }
+        setupLoaderNoneButton(loaderTypePanel);
 
-        if (ConfigManager.getConfigItem("loaders.legacyfabric.enabled", true) == true) {
-            loaderTypePanel.add(loaderTypeLegacyFabricRadioButton);
-        }
-
-        if (ConfigManager.getConfigItem("loaders.quilt.enabled", false) == true) {
-            loaderTypePanel.add(loaderTypeQuiltRadioButton);
-        }
-
-        loaderTypeNoneRadioButton.addActionListener(e -> {
-            selectedLoaderTypeChanged(null);
-        });
-        loaderTypeFabricRadioButton.addActionListener(e -> {
-            selectedLoaderTypeChanged(LoaderType.FABRIC);
-        });
-        loaderTypeForgeRadioButton.addActionListener(e -> {
-            selectedLoaderTypeChanged(LoaderType.FORGE);
-        });
-        loaderTypeLegacyFabricRadioButton.addActionListener(e -> {
-            selectedLoaderTypeChanged(LoaderType.LEGACY_FABRIC);
-        });
-        loaderTypeQuiltRadioButton.addActionListener(e -> {
-            selectedLoaderTypeChanged(LoaderType.QUILT);
-        });
+        setupLoaderQuiltButton(loaderTypePanel);
```

Comment:
Ah this is the issue, this is adding it to the panel before the other, moving this to after everything else brings the order back

Reply:
[3afaf96](https://github.com/ATLauncher/ATLauncher/pull/743/commits/3afaf96a033608028b2171ff4590a5c2f65bf7cb)